Config/Global.xcconfig

```diff
@@ -27,6 +27,9 @@ GCC_C_LANGUAGE_STANDARD = gnu99
 ALWAYS_SEARCH_USER_PATHS = NO
 USER_HEADER_SEARCH_PATHS = "$(SRCROOT)/../MobileCenter/MobileCenter"/**
 
+// :Mark: Code signing
+CODE_SIGN_IDENTITY = 
```

Comment:
We haven't set any code signing configurations for all the projects and just use default settings. However, somehow, all tvOS targets have signing settings unlike iOS and macOS targets so I wanted to set this value.
The reason of having this in Global is that we need to take care of signing soon so we can address everything together.